src/ui/domemittermixin.js

```diff
@@ -254,6 +254,15 @@ function getNodeUID( node ) {
 	return node[ 'data-ck-expando' ] || ( node[ 'data-ck-expando' ] = utils.uid() );
 }
 
+// Checks if given node is native DOM Node.
+//
+// @private
+// @param {Node} node
+// @return {Boolean} True when native DOM Node.
+function isDomNode( node ) {
+	return node && node.nodeType && node.nodeName;
```

Comment:
That's too weak. If we'll implement DOM-like class structure, then it's enough that we had these two (rather popular) properties and it blows up unless `addEventListener` was also defined. So I would also check here the presence of this method.

BTW. Are you sure that all kinds of DOM nodes have the `nodeName` defined? I see they most likely have (https://developer.mozilla.org/en/docs/Web/API/Node/nodeName) but I've never used this property in the wild.